apps/youtube_qa/multi_youtube_video_qa/README.md

```diff
@@ -0,0 +1,24 @@
+# YouTube Question Answering
```

Comment:
It is strange to have an app within an app. A user would be confused. I suggest we have a separate app for this or that we make provide users with the option to add multiple videos in the same app. What do you think? @jarulraj

Reply:
Moved it to a separate app folder for now. We can integrate it into the youtube_qa app in a separate commit.